test/unit/printer/job.test.js

```diff
@@ -53,6 +61,11 @@ describe('job creation', () => {
   it('creates the correct amount of layers', () => {
     expect(LayersMock.createLayer).toHaveBeenCalledTimes(3);
   });
+  it('creates the correct canvas size from cm', () => {
+    let xPixelExpected = Math.round(spec.dpi * spec.size[0] / METRIC_DENOMINATOR);
```

Comment:
I think test should expect concrete value sand not expression or functions that comes from the tested function.

Reply:
ok I changed that